drivers/rkt/driver.go

```diff
@@ -140,16 +141,16 @@ type Config struct {
 
 // TaskConfig is the driver configuration of a taskConfig within a job
 type TaskConfig struct {
-	ImageName        string            `codec:"image"`
-	Command          string            `codec:"command"`
-	Args             []string          `codec:"args"`
-	TrustPrefix      string            `codec:"trust_prefix"`
-	DNSServers       []string          `codec:"dns_servers"`        // DNS Server for containers
-	DNSSearchDomains []string          `codec:"dns_search_domains"` // DNS Search domains for containers
-	Net              []string          `codec:"net"`                // Networks for the containers
-	PortMap          map[string]string `codec:"port_map"`           // A map of host port and the port name defined in the image manifest file
-	Volumes          []string          `codec:"volumes"`            // Host-Volumes to mount in, syntax: /path/to/host/directory:/destination/path/in/container[:readOnly]
-	InsecureOptions  []string          `codec:"insecure_options"`   // list of args for --insecure-options
+	ImageName        string             `codec:"image"`
+	Command          string             `codec:"command"`
+	Args             []string           `codec:"args"`
+	TrustPrefix      string             `codec:"trust_prefix"`
+	DNSServers       []string           `codec:"dns_servers"`        // DNS Server for containers
+	DNSSearchDomains []string           `codec:"dns_search_domains"` // DNS Search domains for containers
+	Net              []string           `codec:"net"`                // Networks for the containers
+	PortMap          hclutils.MapStrStr `codec:"port_map"`           // A map of host port and the port name defined in the image manifest file
```

Comment:
The `rkt` driver simplicity clarifies the changes.  On each occurances we do the following substitutions:

```
map[string]string              ->   hclutils.MapStrStr
NewBlockAttrs(n, "string", b)  ->   NewAttr(n, "list(map(string))", b)
```
(and the same with `map[string]int` interface).

Docker has more fields so the incidental changes make harder to see.